docs/estado-arte.md

```diff
@@ -0,0 +1,52 @@
+# Estado del arte
+En esta nota se va a revisar **el estado del arte**.  
+Objetivos:
+* Buscar *software* similares para compararlo (precios).
+* Buscar tecnologías que puedan servirnos.
+
+## Software similares
+1. [Anaplan](https://www.anaplan.com)    
+    > "Anaplan PlanIQ™ helps users drive accurate decisions with forecasting that leverages powerful, integrated artificial intelligence (AI) and machine learning (ML). Collaborate across your enterprise, working with a single source of truth, updated and calculated in real-time. Test multiple scenarios and quickly operationalize best options."
```

Comment:
Avoid copy-pasting? pull useful info for the project, for our use cases, etc. And if this is not copy-pasted it seems so and it seems TLDR, try to condense better the info